package.json

```diff
@@ -33,6 +33,12 @@
   "browser": {
     "./metasync.js": "./dist/metasync.js"
   },
+  "files": [
+    ".babelrc",
+    "dist/",
```

Comment:
Do we need both published (`dist` and `lib`)? Can't we build `dist` after install?

Reply:
AFAIK we would need to make `babel` our runtime dependency.